ErpFetch.js

```diff
@@ -0,0 +1,70 @@
+"use strict"
+
+class ErpFetch {
+  /**
+   * Fetcher to Odoo Rest API
+   * @param {string} baseUrl base URL of the REST API
+   * @param {string} websiteKey Website Key
+   * @param {object} transport fetch object
+   */
+  constructor(baseUrl, websiteKey, transport) {
+    this.baseUrl = baseUrl
+    this.websiteKey = websiteKey
+    this._fetch = transport || fetch
+  }
+  /**
+   * Make HTTP requests to ressource
+   * @param {String} ressource the resource that you wish to fetch
+   * @param {Object} init custom settings that you want to apply to the request (method, headers ...)
+   * @returns Promise
+   */
+  fetch(ressource, init = {}) {
```

Comment:
```suggestion
  fetch_json(ressource, init = {}) {
```

Since this method essentially does the json conversion, and we may want to preserve access to a lower level raw `fetch`.